test/browser_tests/spec_suites/shared/shared_office.js

```diff
@@ -0,0 +1,22 @@
+var Office = require( '../../page_objects/page_office.js' );
+
+describe( 'The offices template', function() {
+  it( 'should properly load in a browser', function() {
+    var page = new Office();
+    page.get( 'ProjectCatalyst' );
```

Comment:
This line is duplicated for every test. Consider moving into the Page Object method and invoke it in beforeEach function in the spec